scipy/optimize/_highs/cython/src/HConst.pxd

```diff
@@ -6,43 +6,47 @@ from libcpp.string cimport string
 
 cdef extern from "HConst.h" nogil:
 
-    const int HIGHS_CONST_I_INF
-    const double HIGHS_CONST_INF
-    const double HIGHS_CONST_TINY
-    const double HIGHS_CONST_ZERO
-    const int HIGHS_THREAD_LIMIT
-    const bool allow_infinite_costs
-    const string FILENAME_DEFAULT
+    const int HIGHS_CONST_I_INF "kHighsIInf"
+    const double HIGHS_CONST_INF "kHighsInf"
+    const double kHighsTiny
+    const double kHighsZero
+    const int kHighsThreadLimit
```

Comment:
For instance, maybe it's obvious to someone with more experience, but why the changes here?

Reply:
Yep, I can try to describe this a little better.  These fall under the broad category of aligning Cython wrappers with the current HiGHS API.  The quoted string is the literal that Cython will use in generated code and matches the changes to HiGHS' API.  I'm taking advantage of that here as a poor-man's typedef to make fewer changes to the existing pxd files